src/Microsoft.AspNetCore.Routing/Patterns/RoutePatternFactory.cs

```diff
@@ -65,6 +118,23 @@ public static RoutePattern Pattern(string rawText, IEnumerable<RoutePatternPathS
             return PatternCore(null, new RouteValueDictionary(defaults), new RouteValueDictionary(constraints), segments);
         }
 
+        /// <summary>
+        /// Creates a <see cref="RoutePattern"/> from a collection of segments along
+        /// with provided default values and constraints.
```

Comment:
+raw text

Reply:
I left the Raw Text our of these headings because it's not significant (ie, it's only informational).